tests/test_model.py

```diff
@@ -228,6 +228,29 @@ def test_call_concat(model, outputs, expected):
             np.testing.assert_equal(y, expected)
 
 
+@pytest.mark.parametrize(
+    'device, num_workers',
+    [
+        ('cpu', None),
+        ('cuda:0', None),
+        ('cpu', 1),
+        ('cuda:0', 1),
+        ('cpu', 2),
+        ('cuda:0', 2),
+    ]
+)
```

Comment:
If you want to have all possible combinations of two parameters, you can also write it as:

```suggestion
@pytest.mark.parametrize('device', ['cpu', 'cuda:0'])
@pytest.mark.parametrize('num_workers', [None, 1, 2])
```